spec/models/target_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+# Test suite for the Target model
+RSpec.describe Target, type: :model do
+  before(:each) do
```

Comment:
Use `let` or `let!` to create data https://github.com/rootstrap/tech-guides/blob/master/ruby/rspec/style_guide.md#let-blocks